src/overlays/actors/ovl_Bg_Hidan_Kowarerukabe/z_bg_hidan_kowarerukabe.c

```diff
@@ -201,6 +201,7 @@ void func_8088A67C(BgHidanKowarerukabe* this, GlobalContext* globalCtx) {
     Vec3f velocity;
     Vec3f pos;
     s16 arg5;
+    // thisx is not required for matching in this function, but is in BgHidanKowarerukabe_FloorBreak, so done for consistency
```

Comment:
Imo we don't need these comments. Using `thisx` is technically allowed to simplify accesses, so it should just be treated as a normal thing imo.

Reply:
Okay, so leave as thisx, remove comment?

Reply:
yeah